### DISCUSSION TITLE
The possibility to upgrade firmware of NEFIT Enviline IDU Monoblock 5-9 E through BBQkees S3

### DISCUSSION THREAD

**Discussion by ServiceMode** [emsesp/EMS-ESP32#1468] [Ideas]
Is it possible to install an feature to have the possibility to upgrade firmware of a NEFIT Enviline IDU Monoblock 5-9 E through BBQkees EMS Gateway S3 or use the BBQkees EMS Gateway to upgrade the firmware of the Enviline IDU Monoblock in home assistant ?

**Comment by proddy** [comment]
nope, never thought of that. How is it done now?

**Reply by ServiceMode** [reply]
A couple of years ago, they had an IP module that was installed and connected to certain terminals. But the last time that I spoke with a service technician from nefit that they probably don't have them anymore.

**Reply by bbqkees** [reply]
I believe its something that can only be done by a technician. But I cannot find any information on how they then do it.
It is not possible technically to do it via an EMS Gateway.
